### ISSUE TITLE
Docs: Details on authentication with private registries

### ISSUE THREAD

**Issue by jheck88** [openclarity/kubeclarity#245]
Based on the [docs](https://github.com/openclarity/kubeclarity#private-registries-support-for-k8s-runtime-scan) authentication with private registries is doable.  However, nothing explains how or where to set a configmap, or environment variables for registry authentication.  The only notation describes AWS secret auth.  How is this configured?

**Comment by FrimIdan** [comment]
Hi @jheck88,

Kubeclarity is using [k8schain](https://github.com/google/go-containerregistry/tree/main/pkg/authn/k8schain#k8schain) of google/go-containerregistry for authenticating to the registries. So if there is an `imagePullSecret` attached to the pod the authentication should just work.
There are options also to configure [ECR](https://github.com/openclarity/kubeclarity#amazon-ecr)/[GCR](https://github.com/openclarity/kubeclarity#google-gcr) credentials in a dedicated secret if needed.

Can you share what is the private registry that you are using and how K8s is configured to pull images from it?

**Comment by jheck88** [comment]
@FrimIdan I've got an private harbor registry that I use for my clusters.  I'm able to stand up kubeclarity using images reverse-proxied through said private harbor registry, and I've attacked a imagePullSecret to each deployment.  I've also mounted a docker config at `/etc/docker/config.json` and i've tried mounting a kubeclarity auth config at `$HOME/.kubeclarity`.  However with each scenario I'm still seeing the following errors (this is one of many).

```
│ time="2022-11-02T03:35:54Z" level=warning msg="Vulnerabilities scan of imageID \"harbor.internal.com/ext.gcr.io/eticloud/k8sec/grype-server@sha256:82a618c5d77f60d8bcbb80ca6549ef73ec3234059721366f09e │
```
